engine/transaction/record/service.go

```diff
@@ -0,0 +1,19 @@
+package record
```

Comment:
TODO(for the future): think about a better name for this package.

Reply:
🤷